packages/web/package.json

```diff
@@ -38,7 +38,7 @@
     "js:finalize": "cd dist/js && shx mv index.esm.js index.js && shx mv index.esm.js.map index.js.map && shx mv index.esm.d.ts index.d.ts && rm index.umd.js index.umd.js.map index.umd.d.ts",
     "lint": "eslint ./",
     "lint:fix": "yarn lint --fix",
-    "test": "npm-run-all test:prepare css:lint js:lint test:unit:coverage types",
+    "test": "npm-run-all test:prepare css:lint lint test:unit:coverage types",
```

Comment:
Isn't the `css:lint` part of the `lint` already? I saw it in one of your latest fixups, I think.

Reply:
You're right. Updated